Doc/reference/expressions.rst

```diff
@@ -454,9 +454,10 @@ generator.  That generator then controls the execution of the generator
 function.  The execution starts when one of the generator's methods is called.
 At that time, the execution proceeds to the first yield expression, where it is
 suspended again, returning the value of :token:`~python-grammar:expression_list`
-to the generator's caller.  By suspended, we mean that all local state is
-retained, including the current bindings of local variables, the instruction
-pointer, the internal evaluation stack, and the state of any exception handling.
+to the generator's caller, but returns :const:`None` when the optional
+:token:`~python-grammer:expression_list` is not passed.  By suspended, we mean that
+all local state is retained, including the current bindings of local variables, the
+instruction pointer, the internal evaluation stack, and the state of any exception handling.
```

Comment:
```suggestion
to the generator's caller,
or ``None`` if :token:`~python-grammer:expression_list` is omitted.
By suspended, we mean that all local state is
retained, including the current bindings of local variables, the instruction
pointer, the internal evaluation stack, and the state of any exception handling.
```

* Fix grammar
* Use more precise language and conventional phrasing
* Be more concise and readable
* Avoid rewrapping where practical to minimize diffs
* Per [standard convention for these docs](https://devguide.python.org/documentation/markup/#quick-reference)

Reply:
Thanks